index.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable */
+const title = document.querySelector('#inputTitle');
+const author = document.querySelector('#inputAuthor');
+const bookList = document.querySelector('#bookList');// eslint-disable-line no-unused-vars
+const addBtn = document.querySelector('#addBtn');
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+}
```

Comment:
- You need to have class methods to add and remove books. Check if add and espcially remove is working fine, eg, with duplicate title or author (if allowed), the right book is still removed.